src/dodal/devices/eiger.py

```diff
@@ -24,6 +25,10 @@ class EigerTriggerNumber(str, Enum):
 
 
 class EigerDetector(Device):
+    class ArmingSignal(Signal):
+        def set(self, value, *, timeout=None, settle_time=None, **kwargs):
+            return self.parent.stage()
```

Comment:
Must: This creates the class but you don't actually have an instance of the signal